doc/SETUP.md

```diff
@@ -1,24 +1,45 @@
 # How to run Alby in the local development environment
 
+This guide walks you through setting up the Alby Lightning Browser Extension for local development, enabling you to modify, test, and debug it efficiently.
+
+Please go through the [Contribution Guidelines](./CONTRIBUTION.md) before going forward with any development. This helps us keep the process streamlined and results in better PRs.
+
+If you have any setup problems, please ensure you have read through all the instructions have all the required software installed before creating an issue.
+
+## ⚡ Prerequisites
+
+- [Node.js](https://nodejs.org) v16 or newer installed (we run tests with v18)
+- [Yarn](https://yarnpkg.com) v1 or v2 installed
+
+Supported but not required
+
+- [nvm](https://github.com/nvm-sh/nvm#intro)
+
+## 🖥️ OS Requirements
+
+For the best development experience, we recommend using Linux or MacOS, as it offers better compatibility, streamlined workflows, and a consistent development environment. If using Windows, we highly recommend to set up WSL 2 to run a Linux environment smoothly on windows. Install [WSL 2](https://learn.microsoft.com/en-us/windows/wsl/install).
+
 ## 🚀 Quick Start
 
-> For Windows users, please use [WSL 2](https://learn.microsoft.com/en-us/windows/wsl/install).
+This guide helps you run the extension in desktop browsers.
 
-- Install dependencies\
+- We use [Yarn](https://yarnpkg.com/) as package manager. To install all required dependencies, run the following command at the root of the project:
   `$ yarn install`
 
-### 💻 Load extension into browser
+### Run Development Build
 
-> **NOTE:** by default, the extension built this way will talk to the testnet API (which runs under [api.regtest.getalby.com](https://api.regtest.getalby.com)). The testnet API is **not stable** but you can do transactions with testnet sats. In case you want to do run a dev build with the mainnet API, add the following `ALBY_API_URL` environment variable to your command: `$ ALBY_API_URL="https://api.getalby.com" yarn run dev:[chrome|firefox]`
+> To run a dev build with the mainnet API, add the following `ALBY_API_URL` environment variable to your command: `$ ALBY_API_URL="https://api.getalby.com" yarn run dev:[chrome|firefox]`
 
 - Start development build, which will automatically watch for file changes:
 
   - Chrome\
-    `$ yarn run dev:chrome`
+    `$ ALBY_API_URL="https://api.getalby.com" yarn run dev:chrome`
```

Comment:
Is there even a point in having this as an .env variable still? could we default to this value at least?

Reply:
true! currently if Alby_api_url not provided it sets to https://api.regtest.getalby.com  in webpack. which ain't functional anymore i guess. I will also update webpack configs in separate pr and update this again